app/script/conversation/EventMapper.js

```diff
@@ -717,6 +717,11 @@ export default class EventMapper {
 }
 
 function addReadReceiptData(entity, event) {
+  const isFromMe = entity.user && entity.user().is_me;
```

Comment:
This will not work (user will not be set during the lifetime of the event in the `EventMapper`)